code/obj/sawflymisc.dm

```diff
@@ -118,11 +118,11 @@
 
 	//due to not having intent hotkeys and also being AI controlled we only need the one proc
 	harm(mob/target, var/mob/user)
-		user.visible_message("<b class='alert'>[user] [pick(list("gouges", "cleaves", "lacerates", "shreds", "cuts", "tears", "saws", "mutilates", "hacks", "slashes",))] [target]!</b>")
-		playsound(user, "sound/machines/chainsaw_green.ogg", 50, 1)
-		take_bleeding_damage(target, null, 17, DAMAGE_STAB)
-		random_brute_damage(target, 14, FALSE)
-		user.lastattacked = target
+		if(!ON_COOLDOWN(user, "sawfly_attackCD", 1 SECONDS))
```

Comment:
Only 1 second?

Reply:
That's the time between player's attacks, and what I balanced them around on singleplayer